packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-dynamic-intervention-policy-group.vue

```diff
@@ -1,20 +1,97 @@
 <template>
-	<p>
-		Set the <strong>{{ config.intervention?.type }}</strong>
-		<strong>{{ config.intervention?.appliedTo }}</strong> to
-		<strong>{{ dynamicInterventions[0].threshold }}</strong> days when it
-		<strong>{{
-			dynamicInterventions[0].isGreaterThan ? 'increase to above' : 'decrease to below'
-		}}</strong>
-		the threshold value <strong>{{ dynamicInterventions[0].value }}</strong> person.
-	</p>
+	<div class="policy-group">
+		<div class="form-header">
+			<div>
+				<tera-input v-if="isEditing" v-model="knobs.name" placeholder="Policy bounds" />
+				<h6 v-else>{{ knobs.name }}</h6>
+				<i
+					:class="{ 'pi pi-check i': isEditing, 'pi pi-pencil i': !isEditing }"
+					:style="'cursor: pointer'"
+					@click="onEdit"
+				/>
+			</div>
```

Comment:
@blanchco created a light-wrapper component that supposedly does this kind of swapping between readonly-text and text-input.

I cannot recall what is the component here, but you may want to see if it is applicable here.

Reply:
`tera-toggleable-edit.vue`, an example of its use is the the create interventions operator

Reply:
Good shoutout thank you for this!